go/libkb/globals.go

```diff
@@ -176,14 +176,21 @@ func (g *GlobalContext) ConfigureKeyring() error {
 	return nil
 }
 
-func VersionMessage(linefn func(string)) {
+type GitHashGetter interface {
+	Hash() string
+}
+
+func (g *GlobalContext) VersionMessage(linefn func(string)) {
 	linefn(fmt.Sprintf("Keybase CLI %s", VersionString()))
 	linefn(fmt.Sprintf("- Built with %s", runtime.Version()))
+	if h, ok := interface{}(g).(GitHashGetter); ok {
```

Comment:
oh i didn't know about go generate.  so there's a chance we get the wrong answer if we forgot to regenerate?